gulpfile.babel.js

```diff
@@ -267,7 +267,6 @@ gulp.task('metadata', () => {
 gulp.task('default', ['clean'], cb => {
   runSequence(
     ['styles', 'styles-grid'],
-    ['scripts'],
```

Comment:
I suppose to me, it seems quite odd that mocha is now responsible for compiling the src, so the dependency (responsibility) tree is kinda strange (but that just might be me)